addons/website/static/src/js/menu/content.js

```diff
@@ -3,13 +3,216 @@ odoo.define('website.contentMenu', function (require) {
+            kwargs: {
+                context: context
+            },
+        }).then(function (page) {
+            self.page = page[0];
+        }));
+        
+        defs.push(this._rpc({
+            model: 'website.redirect',
+            method: 'fields_get',
+        }).then(function (fields) {
+            self.fields = fields;
+        }));
+          
+        return $.when.apply($, defs);
+    },
+    start: function() {
+        var self = this;
+        var context = weContext.get();
+        this.$(".ask_for_redirect").hide();
+        this.$(".redirect_type").hide();
+        
+        
+        //TODO: remove.html() & set self.var & display in view instead of controller (here)
+        this._getPageDependencies(self.page_id, context)
+        .then(function (dependencies) {
+            var dep = [];
+            _.each(dependencies, function( value, index ) {
+                if (dependencies[index].length > 0) {
+                    dep.push(dependencies[index].length + ' ' + index.toLowerCase() + (dependencies[index].length > 1 ? 's': ''));
+                }
+            });
+            if (dep.length > 0) {
+                $("#dependencies_redirect").html("(used in " + dep.join(", ") + ")");
+            }
+        });
+        
+        
+        var l10n = _t.database.parameters;
+        var datepickersOptions = {
+           minDate: moment({ y: 1900 }),
+           maxDate: moment().add(200, "y"),
+           calendarWeeks: true,
+           icons : {
+               time: 'fa fa-clock-o',
+               date: 'fa fa-calendar',
+               next: 'fa fa-chevron-right',
+               previous: 'fa fa-chevron-left',
+               up: 'fa fa-chevron-up',
+               down: 'fa fa-chevron-down',
+              },
+           locale : moment.locale(),
+           format : time.strftime_to_moment_format(l10n.date_format +' '+ l10n.time_format),
+           //widgetParent : self.$(".o_page_management_page_info_form")
+         };
+         if (self.page.date_publish) {
+            datepickersOptions.defaultDate = self.page.date_publish;
+         }
+         
+         this.$("#date_publish_container").datetimepicker(datepickersOptions);
+         
+
+         return this._super.apply(this, arguments);
+    },
+    save: function(data){        
+        var self = this;
+        var context = weContext.get();
+        var url = $(".o_page_management_page_info #page_url").val();
+        
+        var date_publish = $(".o_page_management_page_info #date_publish").val();
+        if (date_publish != "") {
+            date_publish = time.datetime_to_str(new Date(date_publish));
+        }
+        var params = {
+            id: self.page.id,
+            name: $(".o_page_management_page_info #page_name").val(), 
+            //replace duplicate following "/" by only one "/"
+            url: url.replace(/\/{2,}/g,"/"),
+            is_menu: $(".o_page_management_page_info #is_menu").prop('checked'),
```

Comment:
can we rename:
 o_page_management_page_info
to
 o_page_amanagement_page_info_for_website_page_or_another_thing

stuill 15 char and you overload the 80 char by line ;)